pkg/render/logstorage.go

```diff
@@ -895,9 +895,9 @@ func (es elasticsearchComponent) nodeSets() []esv1.NodeSet {
 	var nodeSets []esv1.NodeSet
 	if nodeConfig.NodeSets == nil || len(nodeConfig.NodeSets) < 1 {
 		nodeSet := es.nodeSetTemplate(pvcTemplate)
-		nodeSet.Name = nodeSetName(pvcTemplate)
-		nodeSet.Count = int32(nodeConfig.Count)
 		nodeSet.PodTemplate = es.podTemplate()
+		nodeSet.Name = nodeSetName(nodeSet.PodTemplate)
```

Comment:
The changes here in logstorage look un-related to the certificate management changes. Why are they in here?